From a88fcecc19ff775690b3b83b9cbab013ac7dff4d Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <pierre-hugues.husson@softathome.com>
Date: Fri, 21 Feb 2020 18:02:41 +0100
Subject: [PATCH 17/25] Add props to control Oppo touchpanel

This is needed because /proc/touchpanel is u:object_r:proc:s0
We can't have our own genfscon on it, because we might hit conflicts
---
 vndk.rc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/vndk.rc b/vndk.rc
index 4111696..db8e470 100644
--- a/vndk.rc
+++ b/vndk.rc
@@ -16,3 +16,9 @@ on property:sys.boot_completed=1
 
 on property:persist.sys.phh.mainkeys=*
     setprop qemu.hw.mainkeys ${persist.sys.phh.mainkeys}
+
+on property:persist.sys.phh.oppo.dt2w=*
+    exec_background u:r:phhsu_daemon:s0 root -- /system/bin/sh -c "echo ${persist.sys.phh.oppo.dt2w} > /proc/touchpanel/double_tap_enable"
+
+on property:persist.sys.phh.oppo.gaming_mode=*
+    exec_background u:r:phhsu_daemon:s0 root -- /system/bin/sh -c "echo ${persist.sys.phh.oppo.gaming_mode} > /proc/touchpanel/game_switch_enable"
-- 
2.17.1

